x-pack/packages/ml/aiops_components/src/progress_controls/use_animated_progress_bar_background.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+
+import { css } from '@emotion/react';
+
+export const useAnimatedProgressBarBackground = (color: string) => {
+  return useMemo(() => {
+    const progressBackground = {
+      background: `repeating-linear-gradient(
+            -45deg,
+            transparent 0 6px,
+            rgba(0, 0, 0, 0.1) 6px 12px
+          ),
+          ${color}`,
+      // 0.707 = cos(45deg)
+      backgroundSize: 'calc(12px / 0.707) 100%,  100% 800%',
+      backgroundPosition: 'inherit',
+    };
+
+    return css({
+      'progress[value]': {
+        display: 'block',
+        width: '100%',
+        WebkitAppearance: 'none',
+        MozAppearance: 'none',
+        appearance: 'none',
+        border: 'none',
+        animation: 'aiopsAnimatedProgress 4s infinite linear',
+
+        '::-webkit-progress-inner-element': {
+          overflow: 'hidden',
+          backgroundPosition: 'inherit',
+        },
+        '::-webkit-progress-bar': {
+          backgroundColor: 'transparent',
+          backgroundPosition: 'inherit',
+        },
+
+        '::-webkit-progress-value': progressBackground,
+        '::-moz-progress-bar': progressBackground,
```

Comment:
if I get it right, `emotion` has an auto-prefixer https://github.com/emotion-js/emotion/issues/1523. 
did you try it?

Reply:
The autocomplete/types for the available attributes still falls back to these webkit/moz based ones. It could be because of the differences in names to make this work. I was able to clean up a little bit the other styles though in [77bca25](https://github.com/elastic/kibana/pull/141770/commits/77bca2551a552a69650d68a2d584d0b1935eb094).